pkg/config/validation/validation.go

```diff
@@ -476,9 +476,25 @@ var ValidateDestinationRule = registerValidateFunc("ValidateDestinationRule",
 		}
 
 		v := Validation{}
-		v = appendValidation(v,
-			ValidateWildcardDomain(rule.Host),
-			validateTrafficPolicy(rule.TrafficPolicy))
+		if features.EnableDestinationRuleInheritance {
+			if rule.Host == "" {
+				if len(rule.Subsets) != 0 {
+					v = appendValidation(v,
+						fmt.Errorf("mesh/namespace destination rule cannot have subsets"))
+				}
+				if rule.TrafficPolicy != nil && rule.TrafficPolicy.Tls != nil {
+					v = appendValidation(v,
+						fmt.Errorf("mesh/namespace destination rule cannot have TLS settings, use peer authentication"))
```

Comment:
Peer authentication describes server side behavior, its not an alternative to destination rule

Reply:
You may want to not use auto mtls, or disable mtls from some namespace or anything else. I would agree you *probably* want auto mtls, but it seems inconsistent to not have tls

Reply:
done